types/feathersjs__errors/index.d.ts

```diff
@@ -1,11 +1,16 @@
 // Type definitions for @feathersjs/errors 3.2
 // Project: http://feathersjs.com/
 // Definitions by: Jan Lohage <https://github.com/j2L4e>
+//                 RazzM13 <https://github.com/RazzM13>
 // Definitions: https://github.com/feathersjs-ecosystem/feathers-typescript
 // TypeScript Version: 2.2
 
 export class FeathersError extends Error {
-    constructor(msg: string | Error, name: string, code: number, className: string, data: any)
+    code: number;
```

Comment:
These should probably be readonly.

Reply:
I've marked all `FeathersError` specific properties as read-only, hopefully this is OK.